wgpu-types/src/lib.rs

```diff
@@ -1984,9 +1996,18 @@ impl TextureFormat {
         let float = TextureSampleType::Float { filterable: true };
         let depth = TextureSampleType::Depth;
 
-        // Color spaces
-        let linear = false;
-        let srgb = true;
+        enum ColorSpace {
```

Comment:
Why not expose this, when we get support for certain hdr color spaces, it will be useful

Reply:
I haven't worked with HDR yet, so didn't consider the way it's to be exported. If it's to be added to this enum - then totally let's expose this as an enum. Maybe in a follow-up?